### DISCUSSION TITLE
Уведомление при действии копирования

### DISCUSSION THREAD

**Discussion by IgorPetrovPRO** [moonshine-software/moonshine#371] [Ideas]
Когда нажимаешь на кнопку https://disk.yandex.ru/d/KCSNJSLsT7Tbzw нет никаких уведомлений, показалось что будет более дружелюбно для пользователей, что бы например появилось уведомление "скопировано"

**Comment by lee-to** [comment]
Реализовано https://github.com/moonshine-software/moonshine/pull/382
